korman/exporter/outfile.py

```diff
@@ -206,9 +273,31 @@ def save(self):
         else:
             self._write_deps()
 
+    @property
+    def super_secure_encryption(self):
+        version = self._version
+        if version == pvEoa:
+            return plEncryptedStream.kEncAes
+        elif version == pvMoul:
+            # trollface.jpg
```

Comment:
It's probably a discussion for another time, but this is likely the best option.  The encryption key is server-specific at the MO:UL subversions, and should be handled by the server operator not the Age builder.

Reply:
Indeed. Currently, I just outright disallowed building a python pak for MOUL/CWE because that engine only loads python.pak. Maybe if we ever get a more distributed system with true multiple shard support....